### PR TITLE
feat(FTL-8311): Add option to set walletURL on CSV issuance

### PR DESCRIPTION
**Pull request by lloydAffinidi**
https://user-images.githubusercontent.com/118436059/206489584-7c7443c8-de00-4947-83b6-e111b18fcc47.mov

### CODE REVIEW

File: src/features/iam/iamHelpers.ts [hunk 1]
```diff
@@ -22,6 +23,16 @@ async function askForProjectId(): Promise<string | undefined> {
   return project?.projectId
 }
 
+async function enterWallet(): Promise<string | undefined> {
```
Comment (vitalyrudenko-affinidi) [inline src/features/iam/iamHelpers.ts:26]: Let's use consistent naming:
```suggestion
async function askForWalletUrl(): Promise<string | undefined> {
```

File: src/features/iam/iamHelpers.ts [hunk 2]
```diff
@@ -22,6 +23,16 @@ async function askForProjectId(): Promise<string | undefined> {
...
+  const wallet = await window.showInputBox({
+    value: 'http://localhost:3000/holder/claim',
```
Comment (vitalyrudenko-affinidi) [inline src/features/iam/iamHelpers.ts:28]: @carlos-affinidi so... what about Affinidi Wallet? Do we need to obscure it now?

File: src/features/iam/iamHelpers.ts (hunk 1 above)
Comment (lloydAffinidi) [reply src/features/iam/iamHelpers.ts]: Renamed

File: src/features/iam/iamHelpers.ts (hunk 2 above)
Comment (lloydAffinidi) [reply src/features/iam/iamHelpers.ts:28]: @vitalyrudenko-affinidi I believe Carlos is OOO today, should we raise this on the Slack thread instead and merge?

**Review by vitalyrudenko-affinidi — APPROVED** [review]

**Pull request merged**